pontoon/base/models.py

```diff
@@ -687,6 +690,9 @@ class Resource(models.Model):
     format = models.CharField(
         "Format", max_length=20, blank=True, choices=FORMAT_CHOICES)
 
+    deadline = models.DateField(blank=True, null=True)
+    priority = models.IntegerField(choices=((i, i) for i in xrange(1, 6)), default=1)
```

Comment:
We should add some meaning ful names, e.g.:

1. Lowest
2. Low
3. Normal
4. High
5. Critical

And default should be 3.